sdk/spring/azure-spring-cloud-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/cloudfoundry/AzureCfService.java

```diff
@@ -38,10 +60,23 @@
 
     private final boolean isAzureProperty;
 
+    /**
+     *
+     * @param azureServiceName The Azure service name.
+     * @param cfServiceName The CF Service name.
```

Comment:
Service to service, and maybe we could call it Cloud Foundry instead of using CF here.